.travis/build-coveralls.sh

```diff
@@ -2,9 +2,10 @@
 set -e
 
 source .travis/common-functions.sh
+source .travis/colors.sh
 
 VERSION=$(./mvnw -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.5.0:exec | tail -1)
-echo "Building PMD Coveralls.io report ${VERSION} on branch ${TRAVIS_BRANCH}"
+echo_yellow "[ INFO] Building PMD Coveralls.io report ${VERSION} on branch ${TRAVIS_BRANCH}"
```

Comment:
I think you should encapsulate the `[ INFO]` prefix + yellow color into a function. Same for errors, you could have `echo_info`, `echo_error`, etc.

Reply:
Will do